doc/tutorial/tutorial.mdk

```diff
@@ -2203,8 +2203,8 @@ it is shadowed by the local `x`. The other cases are straightforward.
 
 > _Note for experts: Because we will only reduce closed expressions,
   where all variables
-  are bound by previous lambdas, we will ever only substitute closed
-  expressions (closed values in fact), and this naive definition of

```

Comment:
This is due to cbv evaluation, right? I found it out of place, and don't think it's relevant to capture-avoiding substitution.